frontend/src/modules/entitydetails/components/Metadata.css

```diff
@@ -49,3 +49,13 @@
     margin: 0;
     padding: 0;
 }
+
+.metadata .resource a:nth-child(2)::before {
+    padding-left: 3px;
+}
+
+.metadata .resource a:nth-child(2)::before {
+    content: "•";
```

Comment:
That's not the right dot. Please use the [middot](https://en.wikipedia.org/wiki/Interpunct) instead.